packages/docs/tsconfig.json

```diff
@@ -7,7 +7,7 @@
     "target": "ESNext",
     "useDefineForClassFields": false,
     "experimentalDecorators": true,
-    "module": "ESNext",
+    "module": "NodeNext",
```

Comment:
Does this make sense? Storybook should work in the browser in the end, Node might be the wrong target?